Update register-devices-partner.md

msbemba opened the pull request:
corrected the column order as it is Device serial number, Manufacturer name, and Device model

msbemba committed:
Update register-devices-partner.md

corrected the column order as it is Device serial number, Manufacturer name, and Device model

msbemba commented:
@JohanFreelancer9 Please copy review

msbemba commented:
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3018

jaimeo commented:
Do not make changes like this without checking the SME. Maybe it's the form that needs to change and not the doc....

jaimeo commented:
@AlexBlain should the article change or is the download form out of date?

msbemba commented:
@jaimeo Thank you. I'll keep that in mind.

AlexBlain commented:
@jaimeo @msbemba this is actually from the Autopilot documentation : https://docs.microsoft.com/en-us/partner-center/autopilot#add-devices-to-a-customers-account
I did not change anything from what is written over there :)

JohanFreelancer9 commented on microsoft-365/managed-desktop/get-started/register-devices-partner.md:
```diff
@@ -41,7 +41,7 @@ Once you have established the relationship with your customers, you can leverage
 > You should have received this .csv file with your device purchase. If you didn't receive a .csv file, you can create one yourself by following the steps in [Adding devices to Windows Autopilot](https://docs.microsoft.com/windows/deployment/windows-autopilot/add-devices#collecting-the-hardware-id-from-existing-devices-using-powershell). The Windows PowerShell script is different from the one use ...
 
-If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or the OEM name, serial number, and model (in that column order), or the Windows Product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 
+If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or  serial number, Manufacturer name, and model (in that column order), or the Windows Product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 
```
```suggestion
If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or serial number, manufacturer's name, and model (in that column order), or the Windows product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 
```

JohanFreelancer9 reviewed (changes requested):
Copy review.

msbemba committed:
Update microsoft-365/managed-desktop/get-started/register-devices-partner.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

jaimeo commented:
Per @AlexBlain, the current article matches the current Autopilot article. So either that article ALSO needs to have the order changed or neither of them does. @msbemba where are you getting the order you are proposing in your edit?

msbemba commented:
@jaimeo I downloaded it from the partner center.

msbemba commented:
> Per @AlexBlain, the current article matches the current Autopilot article. So either that article ALSO needs to have the order changed or neither of them does. @msbemba where are you getting the order you are proposing in your edit?

the downloaded file looks like this 

![image](https://user-images.githubusercontent.com/45987684/95246478-2c53ab80-0832-11eb-996e-6dbf3b71048b.png)

jaimeo committed:
Update register-devices-partner.md

Instead of trying to match what is described elsewhere (which also looks different than the edit proposed here), this now will just link to that source. That way, if they ever change it, we don't have to detect that and try to update it in more than one place.
